src/Core/src/Platform/Windows/Styles/WindowRootViewStyle.xaml

```diff
@@ -3,15 +3,6 @@
 	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	xmlns:maui="using:Microsoft.Maui.Platform">
 
-    <ResourceDictionary.ThemeDictionaries>
```

Comment:
I removed this because we now set the title bar to transparent in `ApplyTitlebarColorPrevalence` if the accent color isn't enabled on title bars. Doing this allows the Mica material to be exposed for the title bar, which matches the Win11 style. I'm not totally sure what it looks like on Win10